biome.json

```diff
@@ -28,8 +28,8 @@
       ".codesandbox/ci.json",
       "build.icon-list.js",
       "**/*.hbs",
-      "docs",
```

Comment:
question (non-blocking): I thought we needed docs for the built website files, but I could be misremembering

Reply:
+1 pretty sure we added this line to fix biome failing on built website files that didn't need to be formatted

Reply:
Likely a dodge rebase, I specifically remember _adding_ this

Reply:
It's actually removing a duplicate from this list. It's on line 10.